dev_scripts/template_pyxis.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python3
+from __future__ import print_function
+import os
+import sys
+from pprint import pprint
+
+PYXIS_SERVER_URL = "https://pyxis.stage.engineering.redhat.com/"
+
+# Allow imports from parent directory.
+sys.path.insert(1, os.path.join(sys.path[0], '..'))
+
+os.environ["FRESHMAKER_DEVELOPER_ENV"] = "1"
+os.environ["FRESHMAKER_CONFIG_FILE"] = os.path.join(sys.path[0], "config.py")
+
+from freshmaker import conf, app
+from freshmaker.pyxis import Pyxis
+
+app_context = app.app_context()
+app_context.__enter__()
+
+if len(sys.argv) != 1:
```

Comment:
Optional: You can just use `argsparse`. It's not much additional code and it's cleaner in my opinion.

Reply:
Maybe I will use `argsparse`, but for now I don't use any arguments so I think I don't need it.